app/R/run_model.R

```diff
@@ -30,32 +30,75 @@
 #' @examples
 #' 
 #' ## make toy duration of hospitalisation (exponential distribution)
-#' r_duration <- function(n = 1) rexp(n, .2)
-#'
-#' x <- run_model(Sys.Date(),
-#'                n_start = 66,
-#'                doubling = c(4.5, 5.1, 6, 4.8),
-#'                duration = 14,
-#'                r_los = r_duration,
-#'                n_sim = 10)
-#' x
-#' plot(x)
+#' r_duration <- function(n = 1) rgeom(n, .2) + 2
```

Comment:
delete this dead code

Reply:
Actually it was meant to be an example, I just forgot to update the comments from ## to #'

Reply:
Fixed as of 119827237a7486b6fa9f2d1160387d637a82e2ea